h/static/styles/topbar.scss

```diff
@@ -41,7 +41,7 @@
 
     & > *:first-child, & > *:last-child {
       line-height: 28px;
-      margin: 0 .72em;
+      margin-right: .72em;
```

Comment:
Yeah, but the reason, I changed this is because it makes the search icon look off on the stream. You can see this live on hypothes.is/stream:
![selection_089](https://cloud.githubusercontent.com/assets/521978/8260357/ec504a04-1677-11e5-8d05-311a6d38b65a.png)

See how the search icon doesn't line up with sort.